app/pb_kits/playbook/pb_owner/_owner.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= content_tag(:div,
+    id: object.id,
+    data: object.data,
+    class: object.classname("pb_owner")) do %>
+      <div class="owner_first">
```

Comment:
We can remove this div wrapper. It's not being used.

Reply:
Removed